packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2182,9 +2182,8 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     _showCaretOnScreenScheduled = true;
     SchedulerBinding.instance!.addPostFrameCallback((Duration _) {
       _showCaretOnScreenScheduled = false;
-      if (_currentCaretRect == null || !_scrollController!.hasClients) {
+      if (!_scrollController!.hasClients || !mounted)
```

Comment:
When set `showCursor` to false `_currentCaretRect` will always be null because it only is assigned when the cursor is painted.

Reply:
It is reasonable to add `!mounted` because it will access the RO below, although I haven't thought of what will happen if I don't.